sheets/1/exercise-3.tex

```diff
@@ -57,32 +75,39 @@
     \end{lstlisting}
-    \item \textbf{[2]} Wenn wir ein Objekt länger als nur einmal brauchen müssen wir uns dieses Merken. Dazu verwenden wir eine Variable. Um eine zu erstellen: schreiben wir zuerst die Klasse zu der das Objekt gehört, gefolgt von dem frei wälbaren Namen, den die Variable haben soll. Mit einem \lstinline{=} weisen wir ein Objekt einer Variable zu, indem wir entweder das \lstinline{new Objekt()} direkt dahinter schreiben oder eine andere
+    \item \textbf{[3]} Wenn wir ein Objekt länger als nur einmal brauchen müssen wir uns dieses Merken. Dazu verwenden wir eine Variable. Um eine zu erstellen: schreiben wir zuerst die Klasse zu der das Objekt gehört, gefolgt von dem frei wälbaren Namen, den die Variable haben soll. Mit einem \lstinline{=} weisen wir ein Objekt einer Variable zu, indem wir entweder das \lstinline{new Objekt()} direkt dahinter schreiben oder eine andere
           Variable die bereits einen Wert hat.
           Wenn wir ein Objekt einem Wert zugewiesen haben, können wir es unter dem Namen der Variable wiederfinden.
           \begin{lstlisting}[title=\textbf{Variablen Syntax}]
-    <Typ> <Name>;
-    <Name> = <Klassen Name>();
-    <Name> = <Klassen Name(<Wert>);
-    <Typ> <Name> = <Klassen Name>();
+    <Typ> <Name>; // Variable wird zur späteren Verwendung erzeugt
+    <Name> = new <Klassen Name>(); //Ein Objekt wird erstellt und einer bereits existierenden Variable zugeweisen.
+
+    <Typ> <Name> = <Klassen Name>(); //Eine Variable wird erstellt und ein Objekt das auch erstellt wird, wird sofort zugeweisen
+
+    <Typ> <Name> = <Name>; /*Eine bestehende Variable wird einer neu erstellten zugeweisen.
+    Das Objekt, das unter dem Namen der alten Variable bekannt war. Ist nun unter beiden Namen bekannt.
+    */
+
+    new <Klassen Name>(<Variablen Name>); // Eine bestehende Variable, wird einem Objekt bei der erstellung mit gegeben.
           \end{lstlisting}
           \begin{lstlisting}[title=\textbf{Beispiel (2/3) Variablen},firstnumber=1,frame=ltr]
             Collector pdfCollector = new Collector();
         \end{lstlisting}
     \item \textbf{[2]} Es wurde ja bereits erwähnt, dass Klassen Operationen haben. Außerdem, dass sie \q{alle} Kommandos beinhalten. Wenn man diese Aufrufen möchte: schreibt man den Namen der Variable mit der wir uns das Objekt gemerkt haben oder auch einfach direkt nachdem wir es erstellen. Dann einen \lstinline{.} gefolgt von dem Namen der Operation und (). Manche geben aber auch ein Objekt zurück, dadurch kann man gleich einen Operator davon Aufrufen oder man speichert sich es, mit einer variable zwischen. Außerdem haben sie manchmal Parameter. Genau wie als wir die Objekte aus den Klassen erstellt haben.
           \begin{lstlisting}[title=\textbf{Kommando Syntax},firstnumber=4]
-        <Operation Name>();
-        new <Typ Name>();
-        new <Typ Name>().<Operation Name>();
+        <Operation Name>(); // ein Operation die zum selben Objekt gehört wird aufgerufen.
```

Comment:
Auf was bezieht sich "zm selben Objekt gehört" genau? Das gehört zu der erklärung von `this` was wir ja größtenteils PSE überlassen wollen. Hier also lieber nur sagen eine Operation in der selben Datei/Klasse wird aufgerufen.